src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
```

Comment:
There is an unnecessary empty line at the beginning of the class definition. Please remove it to comply with the coding standards.